python/src/server.py

```diff
@@ -0,0 +1,53 @@
+# from typing import Any, List, Literal, Optional, Union
+
+# from pydantic import validator
+
+from konan_sdk.konan_service.services import KonanService
+from konan_sdk.konan_service.models import KonanServiceBaseModel
+from konan_sdk.konan_service.serializers import (
+    KonanServiceBasePredictionRequest, KonanServiceBasePredictionResponse)
+
+
+# TODO: [1] Define your request schema
+class MyPredictionRequest(KonanServiceBasePredictionRequest):
+    # Add your predict endpoint input schema here
+    # Follow the convention of <field_name>: <type_hint>
+    # Check https://pydantic-docs.helpmanual.io/usage/models/ for more info
+
+    # Optionally add validators for your features
+    # Follow the pydantic convention
+    # Check https://pydantic-docs.helpmanual.io/usage/validators/ for more info
```

Comment:
Can we change these comments to doc string?

Reply:
Will do